doc/tools/stlink-v21.rst

```diff
@@ -0,0 +1,121 @@
+:orphan:
+
+.. _stlink-v21:
+
+ST-LINK/V2-1
+############
+
+Overview
+********
+
+ST-LINK/V2-1 is a serial and debug adapter built into all Nucleo and Discovery
+boards. It provides a bridge between your computer (or other USB host) and the
+embedded target processor, which can be used for debugging, flash programming,
+and serial communication, all over a simple USB cable.
+
+SEGGER offers a firmware upgrading the ST-LINK/V2-1 on board on the Nucleo and
+Discovery boards. This firmware makes the ST-LINK/V2-1 compatible with J-Link
+OB, allowing users to take advantage of most J-Link features like the ultra fast
+flash download and debugging speed or the free-to-use GDBServer.
+
+More informations about upgrading ST-LINK/V2-1 to JLink or restore ST-Link/V2-1
+fireware please visit: `SEGGER`_
+
+Zephyr supports the following debug tools through ST-LINK/V2-1:
+
+* :ref:`st_ST-LINK/V2-1_stlink`
+* :ref:`st_ST-LINK/V2-1_jlink`
+
+.. _st_ST-LINK/V2-1_stlink:
+
+ST-LINK/V2-1
+************
+
+Flashing
+========
+
+The default debug tool is ST-Link so there doesn't need another params when
```

Comment:
Reword as:

    The default debug tool is ST-Link, so no additional
    flashing or debugging command parameters are required: